pages/linux/head.md

```diff
@@ -8,7 +8,7 @@
 
 - Output the first few bytes of a file
 
-`head -c {{size in kilobytes}} {{filename}}`
+`head -c {{size in bytes}} {{filename}}`
```

Comment:
That wouldn't be as intuitive, IMO. But the token should indeed use underscores as word separators, @Parth-Mehrotra, so I'd suggest `{{size_in_bytes}}`.